packages/grid/x-data-grid/src/tests/columns.DataGrid.test.tsx

```diff
@@ -26,7 +26,7 @@ describe('<DataGridPro /> - Columns', () => {
     </div>
   );
 
-  describe('prop: initialState.columns.columns', () => {
+  describe('prop: initialState.columns', () => {
```

Comment:
```suggestion
  describe('prop: initialState.columns.orderedFields / initialState.columns.dimensions', () => {
```

`columns.columns` was the old name during my column dimension and order PR
But I think we still need to be explicit, because `initialState.columns.columnVisibilityModel` is in another `describe`